### PR TITLE
Enhancement: Info option updated to report the URP isolation interface UUID if present

### PR DESCRIPTION
**Pull request by rozumx**

### CODE REVIEW

**Comment by rozumx** [comment]
retest this please

**Comment by rozumx** [comment]
retest this please

File: src/runtime_src/tools/xclbin/FormattedOutput.cxx [hunk 1]
```diff
@@ -346,7 +346,33 @@ reportXclbinInfo( std::ostream & _ostream,
...
   {
     std::string sUUID = XUtil::getUUIDAsString(_xclBinHeader.m_header.uuid);
-    _ostream << XUtil::format("   %-23s %s", "UUID:", sUUID.c_str()).c_str() << std::endl;
+    _ostream << XUtil::format("   %-23s %s", "UUID (xclbin):", sUUID.c_str()).c_str() << std::endl;
+  }
+
+  {
+    // Find the PARTITION_METADATA section (if there is one)
+    for (auto pSection : _sections) {
+      if (pSection->getSectionKind() == PARTITION_METADATA) {
```
Comment (stsoe) [inline src/runtime_src/tools/xclbin/FormattedOutput.cxx]: Can we do early exit?  that is break and continue to avoid the multiple indentations?

Comment (stsoe) [reply src/runtime_src/tools/xclbin/FormattedOutput.cxx]: The idea was 
```
for (auto pSection : _sections) {
  if (pSection->getSectionKind() != PARTITION_METADATA)
    continue;
  boost::property_tree::ptree ptRoot;
  pSection->getPayload(ptRoot);
  if (!ptRoot.empty()) {
     ptPartitionMetadata = ptRoot.get_child("partition_metadata");
     break;
  }
}
```

I don't think you needed to split the code out of the loop, just 'continue' appropriately or break.

Comment (rozumx) [reply src/runtime_src/tools/xclbin/FormattedOutput.cxx]: Updated the code to follow the pattern as described.  And yes, it is much easier to read.

**Pull request merged**